DodgeballDeathmatch/Assets/Scripts/PlayerController.cs

```diff
@@ -14,22 +14,19 @@ public class PlayerController : MonoBehaviour {
     private Rigidbody2D _rbody;
     public float _movementSpeed;
 
-    private void Start()
```

Comment:
We switching to Egyptian Brackets?

Speaking of which, I should change my functions to PascalCase.

Reply:
It seems to be what was being used the most and I wanted it to look uniform. I was using newlines only because unity defaulted to that. I believe PascalCase is standard for member functions in c# so I suggest changing them.

Reply:
Just changed my functions to PascalCase. I was using vertically-aligned braces since it seems that multiple sources say to do so.

It does look neat for now. However, it seems that most of our code is already using vertical braces. The main exception being the class itself.